src/cli/study-crawl.js

```diff
@@ -94,9 +95,11 @@ function studyCrawlResults(results, specsToInclude) {
         .filter(uniqueFilter);
     var idlNamesIndex = {};
     knownIdlNames.forEach(name => {
-        idlNamesIndex[name] = results.filter(spec => {
+        let list = results.filter(spec => {
             return spec.idl && spec.idl.idlNames && spec.idl.idlNames[name];
         });
+        list = list.filter(spec => isLatestLevel(spec, list));
```

Comment:
How about folding this into the above filter, or at least chaining them to get `const list`?

Reply:
That's not the cleanest code for sure but I cannot chain directly: `isLatestLevel` needs to be called with the already filtered list of specs that define the term to handle cases where an IDL term is defined in level X and is no longer defined in level X+1 (that should not happen often in practice. Actually, that's something that could be worth reporting as a potential anomaly).